ros2topic/ros2topic/api/__init__.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.strategy import NodeStrategy
+
+HIDDEN_TOPIC_PREFIX = '_'
+
+
+def get_topic_names_and_types(*, node, include_hidden_topics=False):
```

Comment:
Only new parameters need to be kwargs to avoid any issues. `node` should always be first, even if new parameters are added. There's no reason to make it kwarg like this imo.

This seems like something we should do everywhere or only when required. As it stands it's in contrast to all of our existing code.